cloudformation/custom_resource/cluster.yaml

```diff
@@ -183,6 +197,8 @@ Resources:
 
           def create_or_update(event):
               properties = event["ResourceProperties"]
+              full_event = json.loads(boto3.client("s3").get_object(Bucket="${PclusterCustomResourceBucket}",Key=event["ResourceProperties"].get("ClusterName")+"/event.json")["Body"].read())
```

Comment:
nit: since you already have
properties = event["ResourceProperties"]
what about key = properties.get("ClusterName")+"/event.json")

Reply:
Done